ab-testing/abTest.ts

```diff
@@ -1,5 +1,24 @@
 import { ABTest } from './types.ts';
 
+/**
+ * Tests are defined here. They will be assigned mvt ranges based on their index in the array, the
+ * size of the test and the number of groups.
+ *
+ * For 100% tests to run concurrently with other tests, they should be assigned to different
+ * test spaces. This means that some users will be in multiple tests at the same time.
+ *
+ * Example:
+ * [Space 0]
+ * - 20% Test	control		MVT 0-99
```

Comment:
So the test spaces allow us to reuse MVT ranges for multiple tests running concurrently, and we're capping to 2 concurrent tests for now by only offering 2 test spaces: 0 and 1?